tc/core/polyhedral/schedule_transforms.cc

```diff
@@ -102,10 +102,10 @@ isl::union_map partialSchedule(
 
 namespace {
 /*
- * If "domain" is any filter, then intersect "domain" with that filter.
+ * If "domain" is a filter, then intersect "domain" with that filter.
  */
-isl::union_set anyFilter(const ScheduleTree* node, isl::union_set domain) {
-  if (auto filterElem = node->elemAsBase<ScheduleTreeElemFilter>()) {
+isl::union_set filter(const ScheduleTree* node, isl::union_set domain) {
```

Comment:
Same naming comment as in some previous commit (`applyFilter`) to avoid confusion with both the tree matcher and  `functional::filter`